changelogs/client_server/newsfragments/1658.clarification

```diff
@@ -0,0 +1 @@
+event_annotations: annotion => annotation
```

Comment:
protip: if, when you open a pull request, you check "Allow edits from maintainers", we can do this sort of thing ourselves rather than having to get you to make a change.